README.md

```diff
@@ -76,17 +76,19 @@ The following tokens are replaced in the `name` parameter:
 - `[contenthash]` the hash of `options.content` (Buffer) (by default it's the hex digest of the `xxhash64` hash)
 - `[<hashType>:contenthash:<digestType>:<length>]` optionally one can configure
   - other `hashType`s, i. e. `xxhash64`, `sha1`, `md4` (wasm version), `native-md4` (`crypto` module version), `md5`, `sha256`, `sha512`
-  - other `digestType`s, i. e. `hex`, `base26`, `base32`, `base36`, `base49`, `base52`, `base58`, `base62`, `base64`
+  - other `digestType`s, i. e. `hex`, `base26`, `base32`, `base36`, `base49`, `base52`, `base58`, `base62`, `base64`, `base64safe`
```

Comment:
I changed it to `base64safe` (without `-`), because regexp in `interpolateName` can be used in many place and I don't want to change it